### PR TITLE
Push generated wasm files in postsubmit

### PR DESCRIPTION
**Pull request by bianpengyuan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by bianpengyuan** [comment]
@PiotrSikora ptal, thanks!

File: scripts/generate-wasm.sh [hunk 1]
```diff
@@ -90,3 +93,24 @@ if [[ ${CHECK_DIFF} == 1 ]]; then
...
+  # Get SHA of proxy repo
+  SHA="$(git rev-parse --verify HEAD)"
+  TMP_WASM=$(mktemp -d -t wasm-plugins-XXXXXXXXXX)
+  trap "rm -rf ${TMP_WASM}" EXIT
+  for i in `find . -name "*.wasm" -type f`; do
+    # Get name of the plugin
+    PLUGIN_NAME=$(basename $(dirname ${i}))
+    # Rename the plugin file and generate sha256 for it
+    WASM_NAME="${TMP_WASM}/${PLUGIN_NAME}-${SHA}.wasm"
+    SHA256_NAME="${TMP_WASM}/${PLUGIN_NAME}-${SHA}.sha256"
```
Comment (PiotrSikora) [inline scripts/generate-wasm.sh]: This should be `filename.sha256`, i.e. `${WASM_NAME}.sha256`.

Comment (bianpengyuan) [reply scripts/generate-wasm.sh]: Done.

File: scripts/generate-wasm.sh [hunk 2]
```diff
@@ -90,3 +93,24 @@ if [[ ${CHECK_DIFF} == 1 ]]; then
...
+    # Get name of the plugin
+    PLUGIN_NAME=$(basename $(dirname ${i}))
+    # Rename the plugin file and generate sha256 for it
+    WASM_NAME="${TMP_WASM}/${PLUGIN_NAME}-${SHA}.wasm"
+    SHA256_NAME="${WASM_NAME}.sha256"
+    cp ${i} ${WASM_NAME}
+    sha256sum "${WASM_NAME}" > "${SHA256_NAME}"
+    
+    # push wasm files and sha to the given bucket
+    gsutil cp "${WASM_NAME}" "${SHA256_NAME}" "${DST_BUCKET}"
```
Comment (PiotrSikora) [inline scripts/generate-wasm.sh]: Do we want to overwrite existing files? If not, then you should verify that the target doesn't already exist (`gsutil stat "${DST_BUCKET}/${WASM_NAME}"`).

Comment (bianpengyuan) [reply scripts/generate-wasm.sh]: Done.

**Pull request merged**